spa_ui/self_service/client/app/config/api.config.js

```diff
@@ -0,0 +1,8 @@
+(function() {
+  'use strict';
+
+  angular.module('app.config')
+    .constant('API_BASE', 'http://localhost:3000')
```

Comment:
on the appliance, Rest API run in the web service worker, so 4000.

Not sure how the above is to run. If the UI running in browser (outside
appliance), we would normally access API via the normal 80, so
https://<appliance>
then the /api would get redirected by apache to the 4000 (or 400x depending
on number of web service worker).

On Thu, Aug 20, 2015 at 9:18 AM, Oleg Barenboim <notifications@github.com>
wrote:

> In spa_ui/self_service/client/app/config/api.config.js
> <https://github.com/ManageIQ/manageiq/pull/3963#discussion_r37527175>:
>
> > @@ -0,0 +1,8 @@
> > +(function() {
> > +  'use strict';
> > +
> > +  angular.module('app.config')
> > +    .constant('API_BASE', 'http://localhost:3000')
>
> @abellotti <https://github.com/abellotti> what is the default port of the
> web service worker?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/3963/files#r37527175>.
>



-- 
*Alberto Bellotti*
abellott@redhat.com